plugin/trino-hudi/src/test/resources/hudi-testing-data/hudi_non_part_cow_cl_sc/.hoodie/.schema/20230815074720567.schemacommit

```diff
@@ -0,0 +1 @@
+{"schemas":[{"max_column_id":13,"version_id":20230815074720567,"type":"record","fields":[{"id":0,"name":"_hoodie_commit_time","optional":true,"type":"string","doc":""},{"id":1,"name":"_hoodie_commit_seqno","optional":true,"type":"string","doc":""},{"id":2,"name":"_hoodie_record_key","optional":true,"type":"string","doc":""},{"id":3,"name":"_hoodie_partition_path","optional":true,"type":"string","doc":""},{"id":4,"name":"_hoodie_file_name","optional":true,"type":"string","doc":""},{"id":5,"name":"rowId","optional":true,"type":"string"},{"id":6,"name":"partitionId","optional":true,"type":"string"},{"id":7,"name":"preComb","optional":true,"type":"long"},{"id":8,"name":"name","optional":true,"type":"string"},{"id":9,"name":"versionId","optional":true,"type":"string"},{"id":10,"name":"toBeDeletedStr","optional":true,"type":"string"},{"id":11,"name":"intToLong","optional":true,"type":"int"},{"id":12,"name":"longToInt","optional":true,"type":"long"},{"id":13,"name":"description","optional":true,"type":"string"}]},{"max_column_id":12,"version_id":0,"type":"record","fields":[{"id":0,"name":"_hoodie_commit_time","optional":true,"type":"string","doc":""},{"id":1,"name":"_hoodie_commit_seqno","optional":true,"type":"string","doc":""},{"id":2,"name":"_hoodie_record_key","optional":true,"type":"string","doc":""},{"id":3,"name":"_hoodie_partition_path","optional":true,"type":"string","doc":""},{"id":4,"name":"_hoodie_file_name","optional":true,"type":"string","doc":""},{"id":5,"name":"rowId","optional":true,"type":"string"},{"id":6,"name":"partitionId","optional":true,"type":"string"},{"id":7,"name":"preComb","optional":true,"type":"long"},{"id":8,"name":"name","optional":true,"type":"string"},{"id":9,"name":"versionId","optional":true,"type":"string"},{"id":10,"name":"toBeDeletedStr","optional":true,"type":"string"},{"id":11,"name":"intToLong","optional":true,"type":"int"},{"id":12,"name":"longToInt","optional":true,"type":"long"}]}]}
```

Comment:
> https://github.com/trinodb/trino/pull/18213#issuecomment-1682783292

Let's remove these file and use product test instead.

Reply:
I have removed the tests for the schema commits.